tests/panic_test.rs

```diff
@@ -21,91 +23,39 @@ fn init_staking(sys: &System) {
         },
     );
 
-    assert!(res.log().is_empty());
+    assert!(res.contains(&(
+        USERS[3],
+        Ok::<StakingEvent, Error>(StakingEvent::Updated).encode()
+    )));
 }
 
 fn init_staking_token(sys: &System) {
-    let st_token = Program::from_file(sys, "./target/fungible_token-0.1.3.wasm");
-
-    let res = st_token.send(
-        USERS[3],
-        InitConfig {
-            name: String::from("StakingToken"),
-            symbol: String::from("STK"),
-            decimals: 18,
-        },
-    );
-
-    assert!(res.log().is_empty());
-
-    let res = st_token.send(USERS[3], FTAction::Mint(100000));
-    assert!(!res.main_failed());
-    let res = st_token.send(
-        USERS[3],
-        FTAction::Transfer {
-            from: USERS[3].into(),
-            to: USERS[0].into(),
-            amount: 100000,
-        },
-    );
-    assert!(!res.main_failed());
-
-    let res = st_token.send(USERS[3], FTAction::BalanceOf(USERS[0].into()));
-    assert!(res.contains(&(USERS[3], FTEvent::Balance(100000).encode())));
+    let mut st_token = FungibleToken::initialize(sys);
 
-    let res = st_token.send(USERS[4], FTAction::Mint(10000));
-    assert!(!res.main_failed());
+    st_token.mint(USERS[3], 100000);
+    st_token.transfer(USERS[3], USERS[0], 100000);
+    st_token.balance(USERS[0]).contains(100000);
```

Comment:
```suggestion
    st_token.mint(USERS[0], 100000);
    st_token.balance(USERS[0]).contains(100000);
```
I think this transfer is redundant, tokens can be minted directly to `USERS[0]`.